src/jest.res

```diff
@@ -471,6 +471,23 @@ module Jest = {
     }
   }
   @val external useRealTimers: unit => unit = "jest.useRealTimers"
+  @val external _setSystemTime: Js.Date.t => unit = "jest.setSystemTime"
+  @val external __setSystemTime: unit => unit = "jest.setSystemTime"
+  @val external _setSystemTime: Js.Date.t => unit = "jest.setSystemTime"
+  @val external __setSystemTime: int => unit = "jest.setSystemTime"
+  @val external ___setSystemTime: unit => unit = "jest.setSystemTime"
+
+  type systemTime = Int(int) | Date(Js.Date.t)
+
+  let setSystemTime = (~now: option<systemTime>) =>
```

Comment:
What will `setSystemTime(~now=None) do? Even Jest's documentation doesn't make this clear, but my guess is that it "unsets" any previously set time, thereby falling back to actual system time? It would be nice if that was made clear.

Also, I think having the argument labelled is unnecessary, and it might be more consistent to use polymorphic variants here.

So to sum up, I'd suggest something like this instead:

```suggestion
  let setSystemTime = ([ #unset | #int(int) | #date(Js.Date.t) ]) =>
```

Reply:
If we call `Jest.setSystemTime()` without an argument then the system time is set to Unix epoch: `1970-01-01T00:00:00.000Z`. 

In that case, `None` and `#unset` might be not apparent - what do you think about `#unixEpoch`?

Reply:
Thank for checking! I'm wondering if we can rely on this continuing to be the case though, since it's not documented, and perhaps shouldn't offer it as an option at all. We could always just add it later, without it being a breaking change, whereas removing or changing it later _would_ be a breaking change.

What do you think?

Reply:
I agree! - especially that resetting time to the UNIX epoch is kind of weird.

Reply:
Also a good point. Thanks! I'll merge and do a release when I get home later today.

Reply:
Thanks!